SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -52,8 +52,7 @@ Conflicts: nvidia-container-toolkit <= 1.10.0-1
 Provides tools such as the NVIDIA Container Runtime and NVIDIA Container Toolkit CLI to enable GPU support in containers.
 
 %prep
-%autosetup -p1
-tar -xvf %{SOURCE1}
+%autosetup -p1 -a1
```

Comment:
🥇

Reply:
TIL about `-a1`! Though I do want to check - @Kanishk-Bansal will this apply the patches first and then extract the additional source archive? or will it extract all the archives first and then automatically apply patches?
My initial searching on this option leads me to believe the former, but we ideally want the latter behavior (though I might be mistaken with my search)

Reply:
Nevermind - I ran this experimentally and found that the right behavior is performed (extract all sources first, then patch). https://dev.azure.com/mariner-org/mariner/_build/results?buildId=786609&view=results